lms/templates/course_modes/choose.html

```diff
@@ -12,6 +12,258 @@
     ${_("Enroll In {course_name} | Choose Your Track").format(course_name=course_name)}
 </%block>
 
+<%block name="testcss">
+<style>
+/* This css was added as part of the LEARNER-1312 test */
```

Comment:
Pretty minor, but s/test/experiment. Test means something else in this context and might confuse someone.